src/application/components/Cache/Cache.tsx

```diff
@@ -12,30 +9,6 @@ import { Loading } from "./common/Loading";
 
 const { Header, Sidebar, Main, Content } = SidebarLayout;
 
-const h1Styles = css`
-  font-family: monospace;
-  font-weight: normal;
-  font-size: ${rem(20)};
-`;
-
-const headerLabelStyles = css`
-  margin: ${rem(3)} 0 0 ${rem(8)};
-  font-family: "Source Sans Pro", sans-serif;
-  color: ${colors.grey.light};
-  text-transform: uppercase;
-  font-size: ${rem(11)};
-`;
-
-const noDataStyles = css`
```

Comment:
To me, this is the biggest advantage of this approach. These `css` blocks take up a lot of vertical space and are pretty disconnected from the component they style, which means a lot of back and forth to figure out the styles applied the element.